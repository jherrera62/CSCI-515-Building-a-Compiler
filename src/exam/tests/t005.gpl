int a=4444;
int q = 0 || 1 ? 99 : 88;
int r = 0 || 0 ? 99 : 88;
int s = -1 ? 99 : 88;
