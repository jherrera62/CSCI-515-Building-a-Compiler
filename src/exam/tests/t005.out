gpl.cpp::main()
  input file(tests/t005.gpl)
  random seed(42)
  read_keypresses_from_standard_input(true)
  dump_pixels(false)
  symbol_table(true)
  graphics(true)

gpl.cpp::main() Calling yyparse()


gpl.cpp::main() after call to yyparse().


Printing the symbol table from main()
int a = 4444
int q = 99
int r = 88
int s = 99
gpl.cpp::main() Calling window->initialize().


gpl.cpp::main() Passing control to window->main_loop().

User quit program. Printing the symbol table.
int a = 4444
int q = 99
int r = 88
int s = 99
